data/snehap02.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Sneha Purkayastha",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Exploring the path of Web Development. Obsessed with tailwindcss, learning Javascript.",
+  "avatar": "https://github.com/snehap02.png",
+  "links": [
+  {
+    "name": "Connect with me on Twitter",
+    "url": "https://twitter.com/SnehaPurkayast8",
+    "icon": "FaTwitter"
+},
+{
+    "name": "Connect with me on LinkedIn"
```

Missing comma  `,` there.
```suggestion
    "name": "Connect with me on LinkedIn",
```